pkgs/development/ocaml-modules/cudf/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib, fetchzip, stdenv, ocaml, ocamlbuild, findlib, ocaml_extlib, glib, perl, pkg-config, stdlib-shims, ounit }:
+
+stdenv.mkDerivation {
+  pname = "cudf";
```

Comment:
Please include the version of OCaml in the `pname`.

Reply:
Done